test_methods.go

```diff
@@ -23,20 +23,30 @@ type TestMockData struct {
 	WithError       error       // Caller's response.
 }
 
+const (
```

Comment:
None of these are used?

Reply:
I used only one for now in tests, maybe it's not the best place to put them. I had in plan to use others since, 404 isn't shouldn't be returned from some calls 
https://github.com/golift/starr/blob/cfcf503e465fee35b62ea82d40062f0b55f15eab/sonarr/series_test.go#L526